src/app/pages/FrontPage/FrontPage.jsx

```diff
@@ -110,8 +113,7 @@ const FrontPage = ({ pageData, mostReadEndpointOverride }) => {
         <IndexPageContainer>
           {groups.map((group, index) => (
             <Fragment key={group.title}>
-              {group.type === 'useful-links' &&
-                renderMostRead(mostReadEndpointOverride)}
+              {group.type === 'useful-links' && renderMostRead(pageData)}
```

Comment:
`renderMostRead` does not need `pageData` as a param

 ```suggestion
              {group.type === 'useful-links' && renderMostRead()}
```